include/vctr/Expressions/BasicMath/Add.h

```diff
@@ -29,11 +29,11 @@ template <size_t extent, class SrcAType, class SrcBType>
 class AddVectors : ExpressionTemplateBase
 {
 public:
-    using value_type = std::common_type_t<ValueType<SrcAType>, ValueType<SrcBType>>;
+    using value_type = CommonVecExpressionType<ValueType<SrcAType>, ValueType<SrcBType>, [] (auto&& a, auto&& b) { return a + b; }>;
```

Comment:
I can reproduce the compile errors locally:
```
C:\Users\frederik.siepe\cleanvoice\ext\pago-modules\ext\VCTR\include\vctr\Expressions\BasicMath\Add.h(32,130): error C2888:
 'auto <lambda_3_>::operator ()(_T1 &&,_T2 &&) const': symbol cannot be defined within namespace 'expressions' [C:\Users\fr
ederik.siepe\cleanvoice\build\cleanvoice.vcxproj]
  (compiling source file '../src/plugin/processor/offline_analysis/deesser/realtime_inference/HybridModel.cpp')
  C:\Users\frederik.siepe\cleanvoice\ext\pago-modules\ext\VCTR\include\vctr\Expressions\BasicMath\Add.h(32,130):
  the template instantiation context (the oldest one first) is
        C:\Users\frederik.siepe\cleanvoice\ext\pago-modules\ext\VCTR\include\vctr\Expressions\BasicMath\Add.h(29,7):
        while compiling class template 'vctr::expressions::AddVectors'
```
The compiler seems to get hung up on the fact that we're in namespace `expressions` and it wants the lambda (?) to be defined in namespace `detail`. I played around a bit, but couldn't make it work so far...

https://learn.microsoft.com/de-de/cpp/error-messages/compiler-errors-2/compiler-error-c2888?view=msvc-170

Reply:
I reworked it by using a static member function instead of a lambda now. Let's see if that works...